### DISCUSSION TITLE
Pesan Tidak Mengirim/Scan QR Gagal/Timed Out

### DISCUSSION THREAD

**Discussion by ekoyuliawan** [kirimwa-id/forum#175] [Tanya Jawab] [answered]
Mohon dibantu Om @rioastamal . Terima kasih.

**Comment by rickyok** [comment]
Sama om, Tolong dibantu, tidak bisa connect in QR

**Reply by mohroyhannahado** [reply]
Sama juga nih, duh gimana ya. Paket berbayarnya juga belum jadi ya

**Comment by ekoyuliawan** [comment]
> Mohon dibantu Om @rioastamal . Terima kasih.

Error code setelah timeout:

_Couldn't log in. Check your phone's Internet connection and scan the QR code again._

**Comment by zonegoh** [comment]
sama mas, bisa dibantu?

**Comment by udien-arrow** [comment]
punya saya juga, scan qrcode error terus

**Comment by yogapamungkas96** [comment]
sama nih

**Comment by rioastamal** [comment]
Mohon maaf lama tidak update. Isu ini akan segera di-investigasi.

**Comment by rioastamal** [comment, marked as answer]
Update telah dirilis, silahkan logout dari linked device lalu scan ulang QR code.

**Reply by ekoyuliawan** [reply]
The bot is now working! Terima kasih mas Rio.

OOT: Untuk paket berbayarnya apakah ada update mas?
